tsconfig.base.json

```diff
@@ -1,9 +1,10 @@
 {
   "compilerOptions": {
     "baseUrl": ".",
-    "target": "es2015",
     "lib": ["es2018", "dom"],
-    "module": "es2015",
+    "target": "es5",
```

Comment:
Do we still need this or was this left over from the zone split memory space discovery for instanceOf usage?

Reply:
I can reverse it. Probably from playing around with stuff trying to get things worse, but i will revert it.